tests/stm32_bootloader/main.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application to activate the STM32 bootloader mode.
+ *
+ * @author      Benjamin Valentin <benpicco@googlemail.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "shell.h"
+#include "shell_commands.h"
+
+extern void usb_board_reset_in_bootloader(void);
+
+static int _bootloader_reset(int argc, char** argv)
+{
+    (void) argc;
+    (void) argv;
+
+    usb_board_reset_in_bootloader();
+
+    return 0;
+}
```

Comment:
Why not add this to `sys/shell/commands`?

Reply:
Will do, although I could also open a separate PR for it - this is useful for all bootloaders that can be triggered by the application.

Reply:
Agreed! :-)